FastSurferCNN/README.md

```diff
@@ -10,20 +10,25 @@ The network was trained with conformed images (UCHAR, 256x256x256, 1 mm voxels a
 The *FastSurferCNN* directory contains all the source code and modules needed to run the scripts. A list of python libraries used within the code can be found in __requirements.txt__. The main script is called __eval.py__ within which certain options can be selected and set via the command line:
 
 #### General
-* --i_dir: Path to the input volume directory (e.g /your/path/to/ADNI/fs60) or 
+* --in_dir: Path to the input volume directory (e.g /your/path/to/ADNI/fs60) or 
 * --csv_file: Path to csv-file listing input volume directories
-* --in_name: name of the MRI_volume (like mri_volume.mgz, __default: orig.mgz__)
+* --t1: name of the MRI_volume (like mri_volume.mgz, __default: orig.mgz__)
 * --conformed_name: name of the conformed MRI_volume (the input volume is always first conformed, if not already, and the result is saved under the given name, __default: orig.mgz__)
 * --t: search tag limits processing to subjects matching the pattern (e.g. sub-* or 1030*...)
-* --o_dir: Path to output directory (where should predictions be saved). Will be created if it does not already exist.
-* --out_name: name of the prediction; the extension determines the output file format, which can be one of: mgz, nii, or nii.gz (__default: aparc.DKTatlas+aseg.deep.mgz__)
-* --log: name of log-file (information about processing is stored here; __default: deep-seg.log__). Saved in the same directory as the predictions.
-* --order: order of interpolation (0=nearest,__1=linear(default)__, 2=quadratic, 3=cubic) for conformation (if not already done).
+* --sd: Path to output directory (where should predictions be saved). Will be created if it does not already exist.
+* --seg_log: name of log-file (information about processing is stored here; If not set, logs will not be saved). Saved in the same directory as the predictions.
+* --strip: strip suffix from path definition of input file to yield correct subject name. (Optional, if full path is defined for --t1)
+* --lut: FreeSurfer-style Color Lookup Table with labels to use in final prediction. Default: ./config/FastSurfer_ColorLUT.tsv
+* --gn: How often to sample from gaussian and run inference on same sample with added noise on scale factor
```

Comment:
This should be removed. Not sure why this is still in here. Is this also present in the run_prediction.py script?

Reply:
Yes they are. All the variables here also seem to still be in use or am I missing something?

Reply:
This one is (or should not) be used in run_prediction.py: --gn: How often to sample from gaussian and run inference on same sample with added noise on scale factor

Reply:
The gn goes as far as being passed from run_prediction.py to inference.py to dataset.py. I'm assuming it should only be deleted int run_predictions.py so it defaults to 0, correct?
[run_prediction.py#L174](https://github.com/Deep-MI/FastSurfer/blob/9c467e27019171ed34ad9daf8384a349473933e4/FastSurferCNN/run_prediction.py#L174) -> [inference.py#L191](https://github.com/Deep-MI/FastSurfer/blob/9c467e27019171ed34ad9daf8384a349473933e4/FastSurferCNN/inference.py#L191) -> [dataset.py#L81](https://github.com/Deep-MI/FastSurfer/blob/9c467e27019171ed34ad9daf8384a349473933e4/FastSurferCNN/data_loader/dataset.py#L81)

Reply:
All of the parts with gn_noise should be removed. All the checks in run_prediction.py, inference.py and dataset.py. This was just for testing an should not be part of the official release